intercom-java/src/main/java/io/intercom/api/UserPermanentDeleteRequest.java

```diff
@@ -0,0 +1,34 @@
+package io.intercom.api;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@SuppressWarnings("UnusedDeclaration")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class UserPermanentDeleteRequest {
+    @JsonProperty("intercom_user_id")
+    private String intercom_user_id;
+    public UserPermanentDeleteRequest() {
+    }
+    public UserPermanentDeleteRequest(String intercom_user_id) {
```

Comment:
Better naming to call `intercom_user_id` => `intercomUserId`?

Reply:
Ah yes thanks for catching that 👍